airbyte-integrations/connectors/source-stripe/source_stripe/source.py

```diff
@@ -65,24 +65,25 @@ def check_connection(self, logger: AirbyteLogger, config: Mapping[str, Any]) ->
     def streams(self, config: Mapping[str, Any]) -> List[Stream]:
         authenticator = TokenAuthenticator(config["client_secret"])
         args = {"authenticator": authenticator, "account_id": config["account_id"]}
+        incremental_args = {**args, "start_date": config["start_date"]}
```

Comment:
why its only for incremental streams? Shouldn't start_date  work with full refresh streams as well?

Reply:
It should be only for incremental streams.